src/dist/nfs/nfs_client_impl.cpp

```diff
@@ -34,11 +34,14 @@
  */
 #include <dsn/utility/filesystem.h>
 #include <queue>
+#include <dsn/tool-api/command_manager.h>
 #include "nfs_client_impl.h"
 
 namespace dsn {
 namespace service {
 
+DSN_DEFINE_int32("nfs", max_copy_rate_megabytes, 500, "max rate of copying from remote node(MB/s)");
```

Comment:
How about simplify to `max_copy_mbps`?

Reply:
`mbps` means `million bits per second`, here is `MB`